Gemfile.lock

```diff
@@ -23,6 +23,7 @@ GEM
     unicode-display_width (2.1.0)
 
 PLATFORMS
+  arm64-darwin-22
```

Comment:
was this intentionally added?

Reply:
Nope, I'm going to remove it. I reverted the change but somehow gets there.

Reply:
Still gets there. I'm going to keep it.

Reply:
It's fine to keep it this change committed.